uxexperience/settings.py

```diff
@@ -106,8 +105,11 @@
 ]
 
 # Heroku DB configuration
-db_from_env = dj_database_url.config(conn_max_age=500)
-DATABASES['default'].update(db_from_env)
+try: import dj_database_url
```

Comment:
it's so I can run it locally without heroku. It just skips importing heroku stuff